process.go

```diff
@@ -2,19 +2,28 @@ package libcontainer
 
 import "io"
 
-// Configuration for a process to be run inside a container.
+// Process specifies the configuration and IO for a process inside
+// a container.
 type Process struct {
 	// The command to be run followed by any arguments.
 	Args []string
+
+	// Env specifies the environment variables for the process.
+	Env []string
+
+	// User will set the uid and gid of the executing process running inside the container
+	// local to the contaienr's user and group configuration.
+	User string
+
+	// Cwd will change the processes current working directory inside the container's rootfs.
+	Cwd string
+
 	// Stdin is a pointer to a reader which provides the standard input stream.
+	Stdin io.Reader
+
```

Comment:
WDYT about also adding capabilities here? There are use cases where one would want to go and tweak a container with additional caps.